ams/server/src/main/java/com/netease/arctic/server/optimizing/TaskRuntime.java

```diff
@@ -272,6 +273,24 @@ public long getTableId() {
     return tableId;
   }
 
+  @Override
+  public String toString() {
```

Comment:
> Should we add a "table#name" field to the TableRuntime object, so that the corresponding table can be easily tracked in the logs? The "tableId" printed in the `toString` method is not convenient enough for problem locating.

Makes sense, but `tableName` cannot be obtained directly from existing variables. To obtain it, we should add the `tableName` field to `TaskDescriptor`.

Reply:
Maybe we can add the table name into the `taskRuntime` class in another PR, this PR looks good enough.